rdrf/registry/groups/registration/base.py

```diff
@@ -0,0 +1,74 @@
+import abc
+
+from rdrf.models.definition.models import Registry
+from django.contrib.auth.models import Group
+from registry.patients.models import Patient, PatientAddress, AddressType
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class BaseRegistration(object):
```

Comment:
Should be `class BaseRegistration(abc.ABC):` and then `BaseRegistration` can't be initiated because it has @abtractmethods .

Reply:
Updated.